tests/v2/actions/clusters/verify.go

```diff
@@ -0,0 +1,39 @@
+package clusters
+
+import (
+	"errors"
+	"net/url"
+
+	steveV1 "github.com/rancher/shepherd/clients/rancher/v1"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	LabelWorker = "labelSelector=node-role.kubernetes.io/worker=true"
+)
+
+var (
+	SmallerPoolClusterSize = errors.New("Machine pool cluster size is smaller than expected pool size")
+)
```

Comment:
I'm not a big fan of using global variables because, when you are looking through the flow of the program, you have to make sure this variable hasn't been modified anywhere.  Also, when this is used in the port_test, it is unclear what this variable contains.

It might be better if you are trying to standardize the error message, to store the message string in a constant, and then use that in the errors.New function. It'll make your intentions more straightforward.